<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeasureSettings</class>
 <widget class="QWidget" name="MeasureSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>284</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>18</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="vLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="hLayoutTitleBar">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lbl">
         <property name="text">
          <string>Measure</string>
         </property>
         <property name="menu_heder_lbl" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="hSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="lblChanName">
         <property name="text">
          <string>Channel</string>
         </property>
         <property name="menu_header_lbl" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="vSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>2</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>2</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid rgb(255, 114, 0);</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>256</width>
          <height>545</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="scopy::gui::SubsectionSeparator" name="widgetMeasurementsSubsSep" native="true"/>
         </item>
         <item>
          <layout class="QGridLayout" name="gLayoutMeasurements">
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>20</number>
           </property>
           <property name="horizontalSpacing">
            <number>40</number>
           </property>
           <property name="verticalSpacing">
            <number>16</number>
           </property>
           <item row="1" column="0">
            <widget class="scopy::gui::CustomSwitch" name="btnMeasDisplayAll">
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="scopy::gui::CustomSwitch" name="btnCounter">
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="lbl_3">
             <property name="text">
              <string>Counter</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QPushButton" name="btnMeasDeleteAll">
             <property name="styleSheet">
              <string notr="true">QPushButton {
  min-height: 30px;
  max-height: 30px;
  min-width: 240px;
  max-width: 240px;

  border: 0px;
}</string>
             </property>
             <property name="text">
              <string>Delete All</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="blue_button" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="lbl_2">
             <property name="text">
              <string>Display All</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="scopy::gui::SubsectionSeparator" name="widgetStatisticsSubsSep" native="true"/>
         </item>
         <item>
          <layout class="QGridLayout" name="gLayoutStatistics">
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>20</number>
           </property>
           <property name="horizontalSpacing">
            <number>40</number>
           </property>
           <property name="verticalSpacing">
            <number>16</number>
           </property>
           <item row="0" column="0">
            <widget class="scopy::gui::CustomSwitch" name="btnStatisticsEn">
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="btnStatisticsReset">
             <property name="styleSheet">
              <string notr="true">QPushButton {
  min-height: 30px;
  max-height: 30px;
  min-width: 100px;
  max-width: 100px;

  border: 0px;

  font-size: 14px;
}</string>
             </property>
             <property name="text">
              <string>Reset</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="blue_button" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QPushButton" name="btnstatsDeleteAll">
             <property name="styleSheet">
              <string notr="true">QPushButton {
  min-height: 30px;
  max-height: 30px;
  min-width: 240px;
  max-width: 240px;

  border: 0px;
}</string>
             </property>
             <property name="text">
              <string>Delete All</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="blue_button" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="scopy::gui::SubsectionSeparator" name="widgetCustomSelectionSubsSep" native="true"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="hLayoutHMeasurements">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
          </layout>
         </item>
         <item>
          <spacer name="vSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="hLayoutVMeasurements">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
          </layout>
         </item>
         <item>
          <spacer name="vSpacer_8">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="scopy::gui::SubsectionSeparator" name="widgetGatingSettingsSubsSep" native="true"/>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="verticalSpacing">
            <number>16</number>
           </property>
           <item row="1" column="0">
            <widget class="scopy::gui::CustomSwitch" name="btnGatingEnable">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="lbl_5">
             <property name="text">
              <string>Enable</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <spacer name="hSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="vSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>197</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>scopy::gui::CustomSwitch</class>
   <extends>QPushButton</extends>
   <header location="global">scopy/gui/custom_switch.hpp</header>
  </customwidget>
  <customwidget>
   <class>scopy::gui::SubsectionSeparator</class>
   <extends>QWidget</extends>
   <header location="global">scopy/gui/subsection_separator.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
